addons/sky_3d/src/Sky3D.gd

```diff
@@ -126,6 +126,13 @@ func show_sky() -> void:
 @export_group("Time")
 
 
+@export_custom(PROPERTY_HINT_NONE, "", PROPERTY_USAGE_EDITOR | PROPERTY_USAGE_READ_ONLY) 
+var game_date: String = ""
```

Comment:
All Sky3D variables need a ## Comment so it shows up in the inspector and in editor help. This isn't important for TOD/Skydome which may get integrated later.